### Python Assignment 18.1

** Problem Statement **

Blood glucose levels for obese patients have a mean of 100 with a standard deviation of
15, A researcher thinks that a diet high in raw cornstarch will have a positive effect on
blood glucose levels. A sample of 36 patients who have tried the raw cornstarch diet
have a mean glucose level of 108. Test the hypothesis that the raw cornstarch had an
effect or not.

#### Step1:
** H0: The Mean blood glucose level of obese patients = 100 <br>
H1: The Mean blood glucose level of obese patients > 100 **

In [2]:
import scipy.stats as stat
import numpy as np

#### Step 2: 
    The significance level = 5%

#### Step 3:
Making a Decision - Compute the z-value

z = (x-mu)/(sigma/sqrt(n))

In [3]:
z = (108-100)/(15/np.sqrt(36))
z

3.2000000000000002

![z-table](ztable.jpg)

You can look at the probability by looking at z- table and p-value associated with 3.20 is 0.9994 i.e. probability of having value less than 108 is 0.9994 and more than or equals to 108 is (1-0.9994)=0.0006.

#### Step-4: Conclusion
It is less than p-value(0.05) so we **will reject the Null hypothesis** i.e. there is raw cornstarch effect.